x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/type/DataType.java

```diff
@@ -44,11 +44,11 @@
     OBJECT(        "object",         JDBCType.STRUCT,    -1,                0,                 0,  false, false, false),
     NESTED(        "nested",         JDBCType.STRUCT,    -1,                0,                 0,  false, false, false),
     BINARY(        "binary",         JDBCType.VARBINARY, -1,                Integer.MAX_VALUE, Integer.MAX_VALUE,  false, false, false),
-    DATE(                            JDBCType.DATE,      Long.BYTES,        24,                24, false, false, true),
+    DATE(                            JDBCType.DATE,      Long.BYTES,        3,                 29, false, false, true),
```

Comment:
The precision here (3) and the comment below (about display size and precision) are incompatible I think.

Reply:
The `3` shows how many second fractional digits are used. currently millis -> 3. This was fixed here: https://github.com/elastic/elasticsearch/commit/1557d77b07bbeef99604857c1ca824e20771c283#diff-a3aa9a4b764c281befa4ee231082c298R51

Reply:
Ok. Thanks. Then the code comment is inaccurate or not necessary?
1. The precision is not 23 + 1 anymore
2. the timezone is not Z only anymore

Reply:
Ah, yes. Thx will fix it.